partcad/src/partcad/provider_factory_store.py

```
@@ -37,6 +37,7 @@ def __init__(self, ctx, source_project, target_project, config):
             self.provider.query_order = self.query_order
 
     async def is_part_available(self, cart_item: ProviderCartItem):
+        pc_logging.debug("Checking availability of %s" % cart_item)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)